proto/accessgraph/v1alpha/query.proto

```diff
@@ -0,0 +1,125 @@
+// Copyright 2023 Gravitational, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package accessgraph.v1alpha;
+
+import "teleport/legacy/client/proto/event.proto";
+import "teleport/legacy/types/types.proto";
+
+option go_package = "github.com/gravitational/teleport/gen/proto/go/accessgraph/v1alpha;accessgraphv1alpha";
+
+// AccessGraphService is a service for interacting the access graph service.
+service AccessGraphService {
+  // Query queries the access graph.
+  // Currently only used by WebUI.
+  rpc Query(QueryRequest) returns (QueryResponse);
+  // GetFile gets a static UI file from the access graph container.
+  rpc GetFile(GetFileRequest) returns (GetFileResponse);
+
+  // SendEvent sends an event to the access graph service.
+  rpc SendEvent(SendEventRequest) returns (SendEventResponse);
+
+  // SendResource sends a resource to the access graph service.
+  rpc SendResource(SendResourceRequest) returns (SendResourceResponse);
+}
+
+// Node is a node in the access graph.
+message Node {
+  // ID is the unique ID of the node.
+  string id = 1;
+  // Kind is the kind of the node, ex: "user", "user_group", "resource", etc.
+  string kind = 2;
```

Comment:
Do we generally make these comments match the casing of the field? like `id` instead of `ID`?

Reply:
I follow the official Proto3 guideline https://protobuf.dev/programming-guides/proto3/
For the new code (that is outside the `api/proto/legacy` `buf` linter forces us to follow this style.

Reply:
FYI, [we do require some comments](https://github.com/gravitational/teleport/blob/74e41d0379fb6daae932d2523e525fbe4244c90e/api/proto/buf.yaml#L12-L15), but technically you don't need to use Go-style in here (unless I'm missing something). Meaning you don't _need_ to start with the field name first.